internal/controller/storageclient_controller.go

```diff
@@ -447,8 +447,17 @@ func (r *storageClientReconcile) deletionPhase(externalClusterClient *providerCl
 		}
 	}
 
+	job := &batchv1.CronJob{}
+	job.Name = fmt.Sprintf("storageclient-%s-status-reporter", utils.GetMD5Hash(r.storageClient.Name)[:16])
+	job.Namespace = r.OperatorNamespace
+	r.log.Info("deleting status reporter job", "StatusReporter", job.Name)
+	if err := r.Client.Delete(r.ctx, job); err != nil {
```

Comment:
better ignore not found error if there is already an event scheduled you'll always fail here.